native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt

```diff
@@ -16,7 +16,7 @@
 package okhttp3
 
 import org.assertj.core.api.AssertionsForClassTypes.assertThat
-import org.junit.jupiter.api.Test
+import org.junit.Test
```

Comment:
Idea 3: Keep using the legacy APIs, instead of upgrading to junit 5 at the same time

Reply:
Definitely. I think vintage is right. I just didn't want to invest further in JUnit 4.x.

Reply:
I think this bumps the priority of the nice PR to split up MockWebServer.  Then follow up with a rip-the-bandaid approach to move us from Junit 4 to Junit 5.

Reply:
Agreed on MockWebServer.

Reply:
https://github.com/square/okhttp/pull/6315 makes me think investing anytime in Junit 4.x here is likely wasted. Going to look at how to progress the MockWebServer split.